test/helpers/config_scanning.rb

```diff
@@ -9,7 +9,7 @@ module ConfigScanning
       # NewRelic::Agent.config into classes as long as we call the variable config
       AGENT_CONFIG_PATTERN       = /config\[:['"]?([a-z\._]+)['"]?\s*\]/
       DEFAULT_VALUE_OF_PATTERN   = /:default\s*=>\s*value_of\(:['"]?([a-z\._]+)['"]?\)\s*/
-      DEFAULT_INST_VALUE_OF_PATTERN   = /:default\s*=>\s*instrumentation_value_of\(:['"]?([a-z\._]+)['"]?,?\s?:?['"]?([a-z\._]+)['"]?\)/
+      DEFAULT_INST_VALUE_OF_PATTERN   = /:default\s*=>\s*instrumentation_value_of\(:['"]?([a-z._]+)['"]?\)|\(:['"]?([a-z._]+)['"]?\s*,\s*:['"]?([a-z._]+)['"]?\)/
```

Comment:
What is this change doing?

Reply:
It's picking up both parameters correctly.  We can have:
```ruby
instrumentation_value_of(:attribute1)
```
or 
```ruby
instrumentation_value_of(:attribute1, :attribute2)
```
Previously, we were not only getting just one of the attributes, it was actually splitting a single value into two values, like `:attribute` and `:1` which was the bug that led to me realizing I coded that Regex incorrectly to begin with.

Reply:
Okay cool, that makes sense. thank you for explaining that!